gamefixes-steam/465840.py

```diff
@@ -0,0 +1,10 @@
+""" 
+The Last Blade
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    util.protontricks('d3dcompiler_46')
+
```

Comment:
After this trailing new line is deleted, I'll merge this ASAP.

Reply:
done